recipes/fio_py/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: fio_py
+  version: 1.0
+
+source:
+  git_url: https://github.com/smithsp/fusion-io.git
+
+build:
+  number: 0
+  skip: True  # [win or py3k]
+
+requirements:
+  host:
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+    - python
+    - lapack
+    - hdf5
+    - make
+    - mpich
+  run:
+    - python
+    - lapack
+    - hdf5
+    - mpi
+    - mpich
+
+test:
+  imports:
+    - fio_py
+
+about:
+  description: 'fio_py is a suite of tools for reading fusion (specifically M3D-C1) code outputs'
+  dev_url: https://github.com/nferraro/fusion-io
+  home: https://github.com/nferraro/fusion-io
+  summary: 'fio_py is a suite of tools for reading fusion (specifically M3D-C1) code outputs'
```

Comment:
Please add the ``license_file``
Please see
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#license-file